backend/src/controllers/sessionsControllers.ts

```diff
@@ -0,0 +1,49 @@
+
+import { Request, Response } from 'express';
+import { createSessionService } from '../services/sessionsServices';
+import jwt from 'jsonwebtoken';
+
+
+export const createSessionController = async(req: Request, res: Response) => {
+
+    try {
+
+        const result = await createSessionService()
+
+        const currentUserAccessToken = result.currentUserAccessToken.access_code
+        const guestUserAccessToken = result.guestUserAccessToken.access_code
```

Comment:
Thi two line are a bit confusing, we are putting an Access Token or an Access Code inside `currentUserAccessToken` and `guestUserAccessToken` ?

Reply:
I had been using token and code interchangeably which is causing this confusion. I got rid of code all together, and am just using token consistently now throughout the app.